sklearn/svm/src/libsvm/svm.cpp

```diff
@@ -518,7 +527,7 @@ double Kernel::k_function(const PREFIX(node) *x, const PREFIX(node) *y,
 			return 0;  // Unreachable 
 	}
 }
-
+BlasFunctions* NAMESPACE::Kernel::m_blas=0;
```

Comment:
This is needed, since static member of a class is different from other class member. You only declare a member instead of defining it without this line of code, which will cause undefined symbol. Please check the error below.
```
pnp@geezer:~/mlpack-benchmarks$ python run.py -c svm.yaml
[INFO] Script: methods/scikit/svm.py
Traceback (most recent call last):
  File "run.py", line 116, in <module>
    run(args.config, args.lib, args.methods, args.loglevel)
  File "run.py", line 71, in run
    module = Loader.ImportModuleFromPath(values["script"])
  File "/home/pnp/mlpack-benchmarks/util/util.py", line 54, in ImportModuleFromPath
    module = imp.load_module(modName, *tup)
  File "/usr/lib/python3.6/imp.py", line 235, in load_module
    return load_source(name, filename, file)
  File "/usr/lib/python3.6/imp.py", line 172, in load_source
    module = _load(spec)
  File "<frozen importlib._bootstrap>", line 684, in _load
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "methods/scikit/svm.py", line 18, in <module>
    from sklearn import svm as ssvm
  File "/usr/local/lib/python3.6/dist-packages/sklearn/svm/__init__.py", line 13, in <module>
    from ._classes import SVC, NuSVC, SVR, NuSVR, OneClassSVM, LinearSVC, \
  File "/usr/local/lib/python3.6/dist-packages/sklearn/svm/_classes.py", line 3, in <module>
    from ._base import _fit_liblinear, BaseSVC, BaseLibSVM
  File "/usr/local/lib/python3.6/dist-packages/sklearn/svm/_base.py", line 8, in <module>
    from . import _libsvm as libsvm  # type: ignore
ImportError: /usr/local/lib/python3.6/dist-packages/sklearn/svm/_libsvm.cpython-36m-x86_64-linux-gnu.so: undefined symbol: _ZN7svm_csr6Kernel6m_blasE
```

Reply:
Ah okay, thank you for the explanation.